components/Article/PayNote.js

```diff
@@ -229,63 +301,62 @@ const TryNoteCta = compose(withRouter)(({ router, darkMode, payload }) => {
   )
 })
 
-const PayNoteCta = ({ variation, payload, position, darkMode }) => {
-  return (
+const PayNoteCta = ({ payNote, payload, darkMode }) =>
+  payNote.cta ? (
     <div {...styles.cta}>
-      {variation.cta === 'try' ? (
+      {payNote.cta === 'trialForm' ? (
         <TryNoteCta darkMode={darkMode} payload={payload} />
       ) : (
-        <BuyNoteCta variation={variation} position={position} />
+        <BuyNoteCta payNote={payNote} payload={payload} />
       )}
     </div>
-  )
-}
+  ) : null
 
 export const PayNote = compose(
   withRouter,
   withInNativeApp,
-  withMemberStatus
+  withMemberStatus,
+  graphql(memberShipQuery)
 )(
   ({
     router: { query },
     inNativeIOSApp,
     isEligibleForTrial,
+    hasActiveMembership,
+    data: { membershipStats },
     seed,
-    trial,
+    tryOrBuy,
     documentId,
     repoId,
-    series,
-    position
+    position,
+    customPayNotes
   }) => {
-    const variation = getPayNoteVariation({
+    const target = {
       inNativeIOSApp,
       isEligibleForTrial,
-      series,
-      trial,
-      seed,
-      query
-    })
-    const lead = (
-      <Translation
-        baseKey={variation.key}
-        position={position}
-        element='title'
-      />
-    )
-    const body = <Translation baseKey={variation.key} position={position} />
+      hasActiveMembership,
+      trialSignup: query.trialSignup,
+      campaignId: query.campaign || query.utm_campaign
+    }
+    const payNote = getPayNote(target, seed, tryOrBuy, customPayNotes)
+
+    if (!payNote) return null
+
+    const positionedNote = payNote[position]
+
```

Comment:
Support only top / bottom paynotes by leaving everything empty and selecting cta none:
```suggestion
  if (!positionedNote.cta && !positionedNote.title && !positionedNote.body) {
    return null
  }
```

Reply:
Currently it renders this green strip:
<img width="1113" alt="Screenshot 2020-02-18 at 00 54 08" src="https://user-images.githubusercontent.com/410211/74692801-306e8280-51e9-11ea-9944-38ae3754ecd8.png">